apps/src/applab/commands.js

```diff
@@ -1789,6 +1789,20 @@ applabCommands.handleReadValue = function(opts, value) {
   }
 };
 
+applabCommands.getList = function(opts) {
+  var onSuccess = handleGetListSync.bind(this, opts);
+  var onError = handleGetListSyncError.bind(this, opts);
+  Applab.storage.readRecords(opts.tableName, {}, onSuccess, onError);
+};
+var handleGetListSync = function(opts, values) {
+  let l = [];
```

Comment:
More descriptive name? (same for x below)

Reply:
whoops, sorry- missed cleaning this up

Reply:
[nit] and newlines between functions